Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -4295,20 +4295,17 @@ void WebPageProxy::sendWheelEvent(WebCore::FrameIdentifier frameID, const WebWhe
     internals().wheelEventActivityHysteresis.impulse();
 #endif
 
-    if (!hasRunningProcess())
-        return;
-
     Ref process = processContainingFrame(frameID);
     if (protectedDrawingArea()->shouldSendWheelEventsToEventDispatcher()) {
         sendWheelEventScrollingAccelerationCurveIfNecessary(frameID, event);
-        process->protectedConnection()->send(Messages::EventDispatcher::WheelEvent(webPageIDInProcess(process), event, rubberBandableEdges), 0, { }, Thread::QOS::UserInteractive);
+        send(Messages::EventDispatcher::WheelEvent(webPageIDInProcess(process), event, rubberBandableEdges));
```

Comment:
cc @achristensen07 @beidson @szewai Doesn't this send the IPC to a potentially different process now when site isolation is enabled? It seems it would always send to the main frame process now instead of the process backing the given frameID.

I'm no expert on site isolation but this looks very suspicious to me.